redbot/core/commands/commands.py

```diff
@@ -815,7 +815,7 @@ async def invoke(self, ctx: "Context"):
                 return await ctx.send_help()
         elif self.invoke_without_command:
             # So invoke_without_command when a subcommand of this group is invoked
-            # will skip the the invokation of *this* command. However, because of
+            # will skip the the invocation of *this* command. However, because of
```

Comment:
Might as well want to remove the extra "the" here.

Reply:
thanks